front_end/audits2/Audits2Panel.js

```diff
@@ -1,43 +1,242 @@
 // Copyright (c) 2016 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
+
+/**
+ * @typedef {{
+ *     lighthouseVersion: ?string,
+ *     generatedTime: !string,
```

Comment:
technically you can drop the `!` for `string` because variables with primitive types are assumed to never be null (being explicit might be code style for devtools though, don't know).